.github/workflows/drip-django.yaml

```diff
@@ -55,6 +55,12 @@ jobs:
     - name: Checking PEP8 code style
       run: |
         pipenv run flake8 --count
+    - name: Checking Black code formatter
+      run: |
+        pipenv run black . --check
+    - name: Check typing
+      run: |
+        pipenv run mypy --namespace-packages --explicit-package-bases --ignore-missing-imports . --exclude docs --exclude setup.py
```

Comment:
I could remove the setup.py and docs/ folder.  The flag `--ignore-missing-imports` is used also by default by pre-commit mirror in Mypy.
The flags `--namespace-packages --explicit-package-bases` are used because the structure of the folder doesn't get drip as a project folder. It throws an error. This could be change by using a specific package name with `-p` flag using it like `-p drip`. But I MUST specify the folders or packages to check with mypy.